cmd/baseserver/README.md

```diff
@@ -24,11 +24,17 @@ Route | Method | Completed | Description
 /sign|POST|✔️|Sign a transaction
 /tx|POST|✖️|Post a transaction to the blockchain
 /seeds/status|GET|✖️|Returns the information on the last seed
+/build/create_role|POST|✔️|Creates a role. Please note that the role MUST be valid hex for example instead of sending "role", send its hex encoded equivalent "726f6c65"
```

Comment:
So we changed this up in https://github.com/tendermint/basecoin/commit/f2adf3645ace974ab7c0ef4957857c33e668e7a1 and the rational is that using hex makes sending content "more secure"/obfuscated as opposed to just sending a string, cc @ethanfrey.